Resources/Prototypes/_NF/Maps/POI/nfsd.yml

```diff
@@ -0,0 +1,22 @@
+- type: gameMap
+  id: nfsd
+  mapName: 'NFSD Outpost'
+  mapPath: /Maps/_NF/POI/nfsd.yml
+  minPlayers: 0
+  stations:
+    nfsd:
+      stationProto: SecurityFrontierOutpost
+      components:
+        - type: StationNameSetup
+          mapNameTemplate: 'NFSD Outpost'
+        - type: StationJobs
+          overflowJobs:
+            - Passenger  # I do not like this, but do not have a better alternative at this time.
+          availableJobs:
+            HeadOfSecurity: [ 1, 1 ]
+            SeniorOfficer: [ 0, 0 ]
+            Brigmedic: [ 0, 0 ]
+            SecurityOfficer: [ 0, 0 ]
+            SecurityCadet: [ 0, 0 ]
```

Comment:
any reason to leave the detective out?

Reply:
Ever just wanna make detectives less forgettable by giving them their own office, then end up forgetting about detectives past that?